articles/creating-windows-csps.md

```diff
@@ -8,19 +10,21 @@ This guide will help you understand the building blocks to crafting CSPs of vary
 
 > In Fleet, Windows CSPs are called "Custom OS settings." Learn more about Custom OS settings [here](https://fleetdm.com/guides/custom-os-settings).
 
-## What is ADMX and why should you care?
+## ADMX
 
-From Microsoft:
+### What is ADMX and why should you care?
 
->“Mobile Device Management (MDM) policy configuration support expanded to allow access of a selected set of Group Policy administrative templates (ADMX policies) for Windows PCs via the Policy configuration service provider (CSP). This expanded access ensures that enterprises can keep their devices compliant and prevent the risk of compromising security of their devices managed through the cloud.”
+>“Mobile Device Management (MDM) policy configurations allow access of a selected set of Group Policy administrative templates (ADMX policies) for Windows PCs via the Policy configuration service provider (CSP). This expanded access ensures that enterprises can keep their devices compliant and prevent the risk of compromising security of their devices managed through the cloud.”
+>
+> -- <cite>[Microsoft Learn](https://learn.microsoft.com/en-us/windows/client-management/understanding-admx-backed-policies)</cite>
```

Comment:
Nice! I like the link.

I like the thinking with `<cite>` here but I think our website won't format this properly. 

I reverted the changes in this section but make "Microsoft" clickable to your link.